lab_3_ann_retriever/main.py

```diff
@@ -104,6 +112,10 @@ def tokenize(self, text: str) -> list[str] | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not isinstance(text, str):
+            return None
+        text = re.findall(r'[a-zа-яё]+', text.lower())
```

Comment:
hm... Can you explain it?

Reply:
I did the same way in the last lab and explained to my mentor